DEVELOPMENT.md

```diff
@@ -115,3 +132,58 @@ bazel build third_party/chromium
 
 # Lastly, put the address returnd by the web server into the Chromium.
 ```
+
+### Debugging Angular UI Tests Locally
+
+tl;dr. Use `ibazel test` for regular work (supports limiting the test amount with
+`fit/fdescribe` and `console.log` usage) and `ibazel run` for karma console breakpoint
+debugging.
+
+1.  Just run all webapp tests. The job stops after finished. `console.log` is not
+    supported. Not handy on debugging.
+
+    ```sh
+    (tf)$ bazel test //tensorboard/webapp/...
+    ```
+
+    It runs the following tests:
```

Comment:
You probably don't need to list these tests. It should change in the future and I doubt we will update the documentation.